profile.bat

```diff
@@ -0,0 +1,10 @@
+@ECHO off
```

Comment:
this file is needed?

Reply:
i would keep this file so everyone can profile the bot without a lot of knowledge